tests/config.tests/blobset_applied.json

```diff
@@ -3,17 +3,17 @@
   "blobset": {
     "blobs": {
       "arbitrary_manufacturer_id": {
-        "phase": "applied",
```

Comment:
Yes but now rename the file too

Reply:
The phase is now final not applied so

mv blobset_applied.json blobset_final.json